final_task/README.md

```diff
@@ -1,3 +1,53 @@
-# Your readme here
-Some text.
-Checkout how to write this file using *markdown*.
+# RSS_READER
+---------------------------------------------------------------------------
+RSS reader is a command-line utility.  
+
+### Usage
+---------------------------------------------------------------------------
+usage: rss_reader.py [-h] [--source SOURCE] [--version] [--json] [--verbose]  
+                     [--limit LIMIT] [--date DATE]  
+
+Pure Python command-line RSS reader.  
+
+optional arguments:  
+ - -h, --help&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*show this help message and exit*  
+ - --source SOURCE&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*RSS URL*  
```

Comment:
Please try to leave a possibility to specify RSS source without `--source` option

Reply:
fixed